dds/DdsDcpsCore.idl

```diff
@@ -272,22 +272,26 @@ module DDS {
       DISALLOW_TYPE_COERCION,
       ALLOW_TYPE_COERCION
     };
-
+    */
+    typedef short TypeConsistencyEnforcementQosPolicyKind_t;
+    const TypeConsistencyEnforcementQosPolicyKind_t DISALLOW_TYPE_COERCION = 1;
+    const TypeConsistencyEnforcementQosPolicyKind_t ALLOW_TYPE_COERCION = 2;
+    /*
```

Comment:
As it stands here, we are leaving the `POLICY_ID` and `POLICY_NAME` constants (below) commented-out.  Should they be included now?  Even if weren't not setting these policies directly, the constants could be useful when integrating with Discovery.

Reply:
I'm still waiting on the tests in this PR to rerun. If something happens there that I have to rework, I think it would be OK to uncomment assuming it wouldn't cause tests to fail or some other large issue (which it shouldn't). If the tests pass, then I'd prefer to do it in my next issue so we can close this issue out.

Reply:
I agree we don't need to do it in this PR.  I marked it "approved."